src/libutil/source-accessor.cc

```diff
@@ -5,6 +5,25 @@ namespace nix {
 
 static std::atomic<size_t> nextNumber{0};
 
+bool SourceAccessor::Stat::isTypeUnknown() {
```

Comment:
```suggestion
bool SourceAccessor::Stat::isTypeUnknown() 
{
```

Formatting nitpick.

`isTypeUnknown()` is a misnomer since we do know the type now. Maybe it should be something like `isNotNARSerialisable()` to denote that it's a type than isn't supported in a NAR file. (Or `isNARSerialisable()` to avoid the negation.)

Reply:
Done, I did try `clang-format` but then it formatted every single file. It seems nix hasn't followed clang-format for some time?

Reply:
@RossComputerGuy we only enforce it for new files.